src/screens/Home.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import logo from 'assets/images/logo.svg';
+
+const HomeScreen = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="app">
+      <header className="app-header">
+        <img src={logo} className="app-logo" alt="logo" />
+        <p>{t('sample_page.message', { codeSample: '<code>src/App.tsx</code>' })}</p>
+        <a className="app-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer" data-test-id="app-link">
```

Comment:
@rosle Should we move this `data-test-id` as a set of constants?
So we can export for using in the test also.

```
export const homeScreenTestIds = {
  appLink: 'app-link',
}
```

Reply:
Yeah we should. May I do this in other components? This one would be deleted later when implementing the homepage. 🙏